src/shell.js

```diff
@@ -53,6 +53,11 @@ var Module = typeof {{{ EXPORT_NAME }}} != 'undefined' ? {{{ EXPORT_NAME }}} : {
 #endif
 #endif // POLYFILL
```

Comment:
Move this line down to after live 59 so that folks can opt out of the polyfill if they want